### PR TITLE
Align codebase with other addons

### PR DESCRIPTION
**Pull request by FinalAngel**

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,27 +1,24 @@
-Copyright (c) 2014 Torchbox Ltd and individual contributors.
+Copyright (c) 2016, Divio AG
```
Comment (kinkerl) [inline LICENSE.txt:1]: 2016?

File: tests/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# other requirements
+djangocms-helper
+tox
+coverage
+# needed to run djangocms-helper
+django-cms
```
Comment (kinkerl) [inline tests/requirements.txt]: this is kinda wrong to require django-cms. there must be a better way?!

File: README.rst [hunk 3]
```diff
@@ -2,239 +2,85 @@
...
-        <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
-        <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-        <!--[if lt IE 9]>
-        <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
-        <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
-        <![endif]-->
+This is a an open-source project. We'll be delighted to receive your
+feedback in the form of issues and pull requests. Before submitting your
+pull request, please review our `contribution guidelines
+<http://docs.django-cms.org/en/latest/contributing/index.html>`_.
```
Comment (kinkerl) [inline README.rst]: django-cms?

**Review by kinkerl — CHANGES_REQUESTED** [review]

File: LICENSE.txt (hunk 1 above)
Comment (FinalAngel) [reply LICENSE.txt:1]: Yeah thats when there was the first commit :)

**Review by evildmp — APPROVED** [review]

**Review by kinkerl — APPROVED** [review]

**Pull request merged**